go.mod

```diff
@@ -19,3 +19,5 @@ require (
 	golang.org/x/tools v0.0.0-20190927052746-69890759d905 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 )
+
```

Comment:
@Athulus : I'd prefer to stay on go 1.12 till 1.14 or later comes out

Reply:
Umm, i just read the change log for 1.12 and 1.13. It looks like it really doesn't matter.